scipy-1.0/paper.tex

```diff
@@ -902,6 +902,53 @@ \subsection*{Maintainers and contributors}
 are selected on a rolling basis from contributors with a significant history of
 high-quality contributions.
 
+\subsection*{Funding}
+
+
+
+%100,000 annual salary, 112 person-years, 414,473 lines of code -> 11,192,622 (https://www.openhub.net/p/scipy/estimated\_cost, which uses the Basic COCOMO model).
+
+The development cost of SciPy is estimated in excess of 10M dollars by
+OpenHub.  Yet, the project is largely unfunded, having been
+developed predominantly by graduate students, faculty, and members of industry
+in their free time. Small amounts of funding have been applied with success:
+some meetings were sponsored by universities and industry, Google's Summer of Code
+program supported infrastructure and algorithmic work, and teaching grant funds were
+used early on to develop documentation.  However, these small amounts of
+sporadic funding are not commensurate with the enormous stack of important scientific
+software, like those that are used to plan national space missions, that rely
```

Comment:
I think this refers to STScI mentioned earlier, but should probably repeat the citation.